Configuration/Console/Commands.php

```diff
@@ -133,8 +133,7 @@
         ],
         'documentation:generatexsd' => [
             'vendor' => 'typo3_console',
-            'controller' => Helhum\Typo3Console\Command\DocumentationCommandController::class,
-            'controllerCommandName' => 'generateXsd',
+            'class' => Helhum\Typo3Console\Command\Documentation\GenerateXsdCommand::class,
```

Comment:
This needs to be updated to the latest registry structure.

Reply:
done